test-utils/throttleMock.js

```diff
@@ -0,0 +1,24 @@
+module.exports = function() {
+  var queue = []
+  return {
+    throttle: function(fn) {
+      var pending = false
+      return function() {
+        if (!pending) {
+          queue.push(function(){
+            pending = false
+            fn()
+          })
+          pending = true
+        }
+      }
+    },
+    fire: function() {
+      queue.forEach(function(fn) {fn()})
+      queue.length = 0
```

Comment:
Note for later: you should change this to the following to avoid bugs:

```js
var tasks = queue
queue = []
tasks.forEach(function(fn) { fn() })
```

That way, you're only running one tick at a time.